flows/field_test.go

```diff
@@ -39,8 +39,17 @@ func TestFieldValues(t *testing.T) {
 	genderVal := fieldVals["gender"]
 	ageVal := fieldVals["age"]
 
-	assert.Equal(t, types.NewXText("Male"), genderVal.ToXValue(env))
+	test.AssertXEqual(t, types.NewXText("Male"), genderVal.ToXValue(env))
 	assert.Nil(t, ageVal.ToXValue(env)) // doesn't have a value in the right type
+
+	test.AssertXEqual(t, types.NewXObject(map[string]types.XValue{
+		"__default__":      types.NewXText("Age: \nGender: Male"),
+		"activation_token": nil,
+		"age":              nil,
+		"gender":           types.NewXText("Male"),
+		"join_date":        nil,
+		"not_set":          nil,
+	}), flows.Context(env, fieldVals))
```

Comment:
Ok now I'm confused, why is age in `__default__` but not the other nil values?

Reply:
Oh because the typed value is nil? Seems like we should just be consistent there no? If `@fields.age` is nil then it shouldn't be in `__default__`

Reply:
Yeah mistake, updated now